### ISSUE TITLE
personal API skips days incorrectly sometimes

### ISSUE THREAD

**Issue by privatemaker** [tracking-exposed/facebook#145]
when accessing the `/daily/` API for Reality Check such as

[https://testing.tracking.exposed/api/v2/personal/token-token-token/daily/3-2]()

the dates I see in the results are like this

```
[{  
    "day": "2019-10-20" ...
},{
    "day": "2019-10-15" ...
},{
    "day": "2019-10-12" ...
}]
```

when i increase the `skip` part of the query from `3-2` to `3-3` such as: 

[https://testing.tracking.exposed/api/v2/personal/token-token-token/daily/3-3]()

i should see the next logical days. but a few times it skips ahead more than one day like

```
[{  
    "day": "2019-10-12" ...
},{
    "day": "2019-10-03" ...
},{
    "day": "2019-10-02" ...
}]
```

do i misunderstand something with query string or is this a bug?

**Comment by vecna** [comment]
* I reviewed the code, and looks ok.
  * the first "3" you can just ignore it, every number < than 3 is considered as 3.
  * I reproduce the bug! with the token you gave me, it is clear between 0-4 and 0-5:

![image](https://user-images.githubusercontent.com/89555/67634054-aad06b00-f8b7-11e9-9a94-5907ce7a9ecb.png)
![image](https://user-images.githubusercontent.com/89555/67634071-cfc4de00-f8b7-11e9-8688-cbcaae366ebc.png)

**Comment by vecna** [comment]
thanks for spotting this. since I refactored/simplify the approach when addressed #140, I left an additional filtering logic in `daily API` which was not necessary and in fact introducing a bug. Online you can find the fixed version.
